spec/index.html

```diff
@@ -748,9 +748,10 @@ <h3>Converting a <a>YAML document</a></h3>
       </p>
 
       <ol>
-        <li>Create an empty <a>map</a> <var>named nodes</var>,
-          used the associate an anchor name with the <a>node</a>
-          on which it is defined.</li>
+        <li>Create an empty <a>map</a> <var>named nodes dictionary</var>,
```

Comment:
Removing the HTML tags —
```
Create an empty map named nodes dictionary,
```
— just doesn't read right to me. I'm sure it's close, but I'm not sure what to change.

Reply:
It is consistent with how such variables are described in JSON-LD 1.1 API. For example, from the [Compaction Algorithm](https://www.w3.org/TR/json-ld11-api/#algorithm-6):

> create a new array <var>compacted types</var> initialized by  ...

If it was <var>x</var> instead of <var>named nodes dictionary</var>, would it sound any different?

> Create an empty `map` <var>x</var> used the associate an anchor name with the `node`.

Would changing it to "empty `map` named <var>named nodes dictionary</var>" be better? To me _named nodes dictionary_ is the variable, it's not really the name of the variable.